package.json

```diff
@@ -28,7 +28,7 @@
     "build:windows": "webpack --config ./webpack/prod.config.js && set NODE_ENV=production && npm start",
     "build:nostart": "webpack --config ./webpack/prod.config.js",
     "build:langs": "babel plugins/intl/translate.js | node",
-    "dev": "node ./webpack/dev-server.js",
+    "dev": "NODE_ENV=test node ./webpack/dev-server.js",
```

Comment:
This should change, and replace it with `NODE_ENV=development` and also fix the check in email.js. Typically `test` is for `npm test`, not development server. Another alternative would be to remove it all together, and just assume development is the default if NODE_ENV is unset